test/razorTests/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/README.md

```diff
@@ -0,0 +1,50 @@
+﻿# VSCode TextMate Grammar Tests
+
+[Jest](https://jestjs.io/en) is used for VS Code TextMate Grammar unit tests.
+
+## Setup
+
+```bash
+yarn install
```

Comment:
Is this applicable?  Do these actually require yarn?  We definitely should not be introducing another build system when we're already using npm/gulp.